docs/reference/user/provisioning/scim-via-curl.md

```diff
@@ -0,0 +1,172 @@
+# Using the SCIM API with curl {#RefScimViaCurl}
+
+_Author: Matthias Fischmann_
+
+---
+
+This page shows you how to communicate with the wire backend through
+the [SCIM API](http://www.simplecloud.info/) by example.  All examples
+are [curl](https://curl.haxx.se/) (in bash syntax).
+
+If you want to dive into the backend code, start [reading here in our
+backend](https://github.com/wireapp/wire-server/blob/develop/services/spar/src/Spar/Scim.hs)
+and [our hsim library](https://github.com/wireapp/hscim).
+
+
+## Creating an SCIM token
+
+First, we need a little shell environment:
+
+```
+export WIRE_BACKEND=prod-nginz-https.wire.com
+export WIRE_ADMIN=...
+export WIRE_PASSWD=...
+```
+
+SCIM currently supports a variant of HTTP basic auth.  In order to
+create a token in your team, you need to authenticate using your team
+admin credentials.  The way this works behind the scenes in your
+browser or cell phone, and in plain sight if you want to use curl, is
+you need to get a wire token.
+
+```
+export BEARER=$(curl -X POST \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  -d '{"email":"'"$WIRE_ADMIN"'","password":"'"$WIRE_PASSWD"'"}' \
+  https://$WIRE_BACKEND/login'?persist=false' | jq -r .access_token)
```

Comment:
> `login'?persist=false'`

How does this work?

Reply:
I meant the single quotes

Reply:
they keep the `?` from making bash try to expand this into a filename with one (or optionally none?  not sure about the precise semantics.) character intsead of the `?`.